script/releases/v0.5.1-rewardsv2/upgrade.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "rewards-v2",
+    "from": "~0.0.0",
+    "to": "0.5.1",
```

Comment:
Just double checking that this looks correct in terms of versioning.

Reply:
@ypatil12 but yes i believe rewards is guaranteed 0.5.1

Reply:
we were 0.4.3 for PI, so we should be 0.5.1 for V2. slashing is 1.0.0